docs/references/javascript/user/totp.mdx

```diff
@@ -3,79 +3,87 @@ title: Time-based One-time Password
 description: Learn about the methods on the User object that allow you to generate and verify TOTP secrets, disable TOTP, and manage backup codes for user authentication.
 ---
 
-# Time-based one-time password (TOTP)
+# Time-based One-time Password (TOTP)
 
-There are a number of methods on the [`User`](/docs/references/javascript/user/user) object that are related to Time-based One-time Password (TOTP) functionality. These methods allow you to generate and verify TOTP secrets, disable TOTP, and manage backup codes for user authentication.
+Clerk supports multi-factor authentication (MFA) using Time-based One-time Password (TOTP) as a second factor. TOTP is a widely used standard for generating one-time passwords that are valid for a short period of time.
 
-## Methods
+These methods on the [`User`](/docs/references/javascript/user/user) object are related to TOTP functionality and allow you to generate and verify TOTP secrets, disable TOTP, and create backup codes for user authentication.
 
-### `createTOTP()`
+To see a complete example of how to use these methods, see the [TOTP example](#totp-example) section.
 
-```typescript
-function createTOTP(): Promise<TOTPResource>;
-```
+<Callout type="warning">
+  Multi-factor authentication must be enabled in your Clerk settings for these methods to work. See the [Multi-factor authentication](/docs/authentication/configuration/sign-in-sign-up-options#set-up-multi-factor-authentication) section to learn more.
+</Callout>
+
+## `createTOTP()`
 
 Generates a TOTP secret for a user that can be used to register the application on the user's authenticator app of choice. Note that if this method is called again (while still unverified), it replaces the previously generated secret.
 
+```typescript
+function createTOTP(): Promise<TOTPResource>;
+```
 
-#### Returns
+### `createTOTP()` returns
 
 | Type | Description |
 | --- | --- |
-| <code>Promise\<[TOTPResource](#totp-resource)\></code> | The newly created TOTP object. |
+| <code>Promise\<[TOTPResource](#totp-resource)\></code> | This method returns a `Promise` which resolves with the newly created TOTP object. |
```

Comment:
```suggestion
| <code>Promise\<[TOTPResource](#totp-resource)\></code> | This method returns a `Promise` which resolves to the newly created TOTP object. |
```

In the other PRs we've been saying "resolves to" instead of "resolves with"